### PR TITLE
Yet Another OSL Fix

### PR DESCRIPTION
**Pull request by luisbarrancos**
This should fix ticket #1176.

### CODE REVIEW

File: src/appleseed.shaders/src/maya/as_maya_areaLight.osl [hunk 1]
```diff
@@ -106,6 +106,19 @@ surface as_maya_areaLight
     ]],
 )
 {
+    out_center = point("center", 0, 0, 0);
```
Comment (dictoon) [inline src/appleseed.shaders/src/maya/as_maya_areaLight.osl]: Shouldn't that be `point("shader", 0, 0, 0)`?

**Comment by est77** [comment]
I'm wondering if we can remove all the output parameters.
We are not using them and we are not connecting them to other shaders.
It's almost sure that they will be optimized away by OSL.

File: src/appleseed.shaders/src/maya/as_maya_areaLight.osl (hunk 1 above)
Comment (luisbarrancos) [inline src/appleseed.shaders/src/maya/as_maya_areaLight.osl]: @dictoon it should actually be *shader* space, but there is no *shader* space anymore, so the best we can do is to use *object* space, and center will be (0,0,0) of that space. Addressed with 2cfe7f799ec1a8c9fc785af6c52bfc1b252bb62e

**Comment by luisbarrancos** [comment]
@est77 if they will be optimized away unless connected, why would you want to remove Maya features? You can use the center position, to connect to a say, distance node to drive a color or other quantity.
There are much more parameters in the node(s), but they are hidden, and most are pretty useless for us.

screenshot_2016-12-26_08-49-34

**Pull request merged**